lib/custom/setup/FEUsersAddSiteIdTypo3.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @license LGPLv3, http://opensource.org/licenses/LGPL-3.0
+ * @copyright Aimeos (aimeos.org), 2017
+ */
+
+
+namespace Aimeos\MW\Setup\Task;
+
+
+/**
+ * Adds siteid column to fe_users table.
+ */
+class FEUsersAddSiteIdTypo3 extends \Aimeos\MW\Setup\Task\Base
+{
+	private $mysql = array(
+		'fe_users' => array (
+			'ALTER TABLE "fe_users" ADD "siteid" INTEGER NULL'
+		),
+	);
+
+
+	/**
+	 * Returns the list of task names which this task depends on.
+	 *
+	 * @return array List of task names
+	 */
+	public function getPreDependencies()
+	{
+		return [];
+	}
+
+
+	/**
+	 * Returns the list of task names which depends on this task.
+	 *
+	 * @return string[] List of task names
+	 */
+	public function getPostDependencies()
+	{
+		return array('TablesCreateTypo3');
+	}
+
+
+	/**
+	 * Executes the task for MySQL databases.
+	 */
+	protected function mysql()
```

Comment:
This is deprecated. You should use `migrate()` instead:
https://aimeos.org/docs/Developers/Library/Setup_tasks/Targets#Migrate